modules/cstrike/cstrike/CstrikeNatives.cpp

```diff
@@ -1944,17 +1944,66 @@ static cell AMX_NATIVE_CALL cs_get_weapon_info(AMX* amx, cell* params)
 	return 0;
 }
 
-// native cs_get_user_weapon_entity(playerIndex);
+// native cs_get_user_weapon_entity(playerIndex, weaponIndex = CSW_NONE);
```

Comment:
I would prefer to use `weaponId`. It confuses me with the weapon entity index. 
In the code below as well.